nrf_wifi/hw_if/hal/inc/fw/phy_rf_params.h

```diff
@@ -88,7 +70,259 @@
 #define NRF_WIFI_VBAT_LOW  (12)  /* Correspond to (2.5+12*0.07)=3.34V */
 #define NRF_WIFI_VBAT_HIGH (14) /* Correspond to (2.5+14*0.07)=3.48V */
 
+/* Package independent params */
+
+/** Power detector adjustment value. */
+#define PD_ADJUST_VAL 0
+
+/** RX gain adjustment value for both 2.4GHz and 5 GHz bands */
+#define RX_GAIN_OFFSET_LB_CHAN      0
+#define RX_GAIN_OFFSET_HB_LOW_CHAN  0
+#define RX_GAIN_OFFSET_HB_MID_CHAN  0
+#define RX_GAIN_OFFSET_HB_HIGH_CHAN 0
```

Comment:
Suggest not to try and vertically align the values. Specify the value by just leaving a space after the macro name. This will make it easier for maintenance. (General comment for all #defines which have been added below too)